lib/gui/components/safe-webview.js

```diff
@@ -198,26 +199,29 @@ class SafeWebview extends react.PureComponent {
    * @param {Event} event - event object
    *
    * @example
+   *
+   * // In the webview
    * console.log({
-   *   type: 0,
-   *   message: 'Hello, World!'
+   *   command: 'error',
+   *   data: 'Good night!'
    * });
+   *
+   * // or with robot
+   * robot.printError('Good night!');
    */
   static consoleMessage(event) {
-    const LOG_LEVEL = 0;
-    const WARN_LEVEL = 1;
-    const ERROR_LEVEL = 2;
+    const {
+      command,
+      data
+    } = _.attempt(robot.parseMessage, event.message);
```

Comment:
Although according to https://github.com/resin-io/etcher/blob/master/lib/shared/robot/README.md you're 'supposed' to use `robot.getCommand(message)` and `robot.getData(message)` ;-)
And it looks like you can use `robot.isMessage` to avoid using `_.attempt`.